docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,138 @@
+-  Post your GitHub name in the [**#github-access-request**](https://discord.com/channels/1068260532806766733/1078124139983945858) channel and we’ll add you to our [GitHub](https://github.com/codeforpdx/PASS). We keep track of technical and non-technical tasks in [GitHub Projects](https://github.com/orgs/codeforpdx/projects/3).
+- Introduce yourself and post your email in the [**#pass**](https://discord.com/channels/1068260532806766733/1075285803137257544) channel so we can add you to our [Google Drive](https://drive.google.com/drive/u/0/folders/1zTEd34K7Eg7rvg71zS6Uzbwrsct2Lx9E?ths=true). You can also request access privately from Flo -- florian@codeforpdx.org.
+-  Pass is composed of three teams: UX/UI, Project Management, and Developers (back and front end). We are currently meeting in separate groups based on team member availability. The development team meets virtually every Tuesday at 7pm via Google Meet. Bi-weekly in person full group meetings are also back in session!!(details in discord)
+-  Reach out to Flo (Project Manager) on Discord or florian@codeforpdx.org to introduce yourself and share how you wish to contribute.
+-  Our developers will occasionally hang out in the [PASS voice chat](https://discord.com/channels/1068260532806766733/1106779713793433730) on Discord while working on the project. If you ever see people in there, feel free to hop in and ask questions. Consider it office hours.
+
+## 5. Development Process
+
+- ## Choosing an Issue
+1. Search through open [issues](https://github.com/codeforpdx/PASS/issues)
+2. Find an issue that interests you
+3. Check if someone is currently working on that issue and if a [pull request](https://github.com/codeforpdx/PASS/pulls) exists.
+4. Assign yourself to issue and leave a comment stating your intentions.
+- ## Clone the repo
+
+1. Clone repo to local environment in IDE of choice. If you are new to Git/GitHub ou can also check out [this article](https://www.digitalocean.com/community/tutorials/how-to-create-a-pull-request-on-github) for a broad overview.
+   - Open terminal & change working directory to the location you want the repository cloned to.
+   - `git clone https://github.com/codeforpdx/PASS.git` [learn more about git clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository?platform=linux)
+   - `git remote add origin https://github.com/codeforpdx/PASS.git` [learn more about git remote](https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories)
+
+2. Setup instructions to locally run PASS can be found in the [readme](../README.md)
```

Comment:
I believe the link here refers to the more detailed instructions for running PASS locally on `/docs/README.md` rather than the short version on `README.md`?

Reply:
/docs/README.md has the instructions as well but they are sort of hidden amongst many other. Maybe making the two uniform or removing them from /docs/README and keeping it in the main README? Tim had made a previous comment about the importance of having those instructions in the main README and it seems to be common practice

Reply:
The `README.md` in `/docs` are there to be used as the main page for `JSDoc`. Think we should keep what we have there.

We could duplicate what we have there for the instructions onto the main `README.md`

Reply:
resolved in discord VC, linked in main README